posts/2021/01/enhancements-to-the-guvectorize-decorator.md

```diff
@@ -0,0 +1,151 @@
+<!--
+.. title: Enhancements to the guvectorize decorator
```

Comment:
The title is a bit cryptic. How about "Enhancements to Numba's guvectorize decorator" ?